src/assets/google-script/google-script.ts

```
@@ -2,50 +1,150 @@
+    }
+
+    if (this.currentLanguage === language && this.lastLanguage) {
+      return Promise.resolve();
+    }
+
+    if (this.scriptLoaded && this.currentLanguage === language && this.mapReadySubject.value) {
+      return Promise.resolve();
+    }
+
+    // eslint-disable-next-line
+    if (!!this.loadMutex) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)